pinakes/settings/defaults.py

```diff
@@ -413,3 +423,21 @@
 CORS_ALLOW_CREDENTIALS = False
 CSRF_TRUSTED_ORIGINS = env.list("PINAKES_CSRF_TRUSTED_ORIGINS", default=[])
 SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+
+# Metrics collections
+PINAKES_INSIGHTS_TRACKING_STATE = env.bool(
+    "PINAKES_INSIGHTS_TRACKING_STATE", False
+)
+PINAKES_INSIGHTS_URL = env.str("PINAKES_INSIGHTS_URL", default="ingress_url")
+PINAKES_INSIGHTS_USERNAME = env.str(
+    "PINAKES_INSIGHTS_USERNAME", default="ingress_username"
+)
+PINAKES_INSIGHTS_PASSWORD = env.str(
+    "PINAKES_INSIGHTS_PASSWORD", default="ingress_password"
+)
+PINAKES_ANALYTICS_LAST_GATHER = env.str(
```

Comment:
@hsong-rh how would this be set. if the process were to terminate and restart would the env var exist.

Reply:
They are outdated and removed